src/app/hooks/useReCaptcha.ts

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+
+export const useReCaptcha = () => {
+  const [key, setKey] = useState('');
+  const [token, setToken] = useState('');
+
+  const handleLoaded = () => {
+    const windows: any = window;
+
+    windows.grecaptcha.ready((_: any) => {
```

Comment:
you have 2 `any` types here, is there a reason for this?

Reply:
I can type the second one as a string. The first one is because I don't really know what it is. This is for testing purposes that I can only do on staging. After collecting the data, I'll have to refactor and type correctly (and probably beautify) this hook.

Reply:
the type of window is this one `Window & typeof globalThis` if you're extending window you can add that type adding something like this `& CustomProps`